### PR TITLE
pass options to the plugin function

### PR DESCRIPTION
**Pull request by Mariam13**

### CODE REVIEW

File: js/cc_script.js [hunk 1]
```diff
@@ -6,149 +6,137 @@
...
-					slider.animate({"left": leftMargin + clickStep+"px"},clickTimer);
-			}
-			else{
-				slider.animate({"left": "0px"},clickTimer);
+	/*==================================main functions for click====================*/
 		
+					
+		$(".leftDirection").bind(//function for left navigational button mousedown
+			"mousedown", function(){
+				defaults.slider.stop();
```
Comment (hackash) [inline js/cc_script.js]: @Mariam13  settings should be used here

File: js/cc_script.js [hunk 2]
```diff
@@ -6,149 +6,137 @@
  */
 
 (function($){
-	$.fn.ccCarousel = function() {
+	$.fn.ccCarousel = function(options) {
```
Comment (hackash) [inline js/cc_script.js]: check for options to be an object

File: js/cc_script.js [hunk 3]
```diff
@@ -6,149 +6,137 @@
...
- 
\ No newline at end of file
+		
+		/*=========================================part for Images that contain links==================================================*/
+		
+		
+		$(".slider img").on(//function that controls links
+			"click", function(){
+				if($(this).attr("rel")==1){
+					window.open($(this).attr("alt"),"_blank");
```
Comment (hackash) [inline js/cc_script.js]: @Mariam13 please make window local variable by passing it to IIFE

File: js/cc_script.js [hunk 4]
```diff
@@ -5,12 +5,15 @@
  * Date 09.12.11
  */
 
-(function($){
+(function(window,$){
 	$.fn.ccCarousel = function(options) {
 
-	/*================================Configurations==============================*/
+		if (typeof options !== Object) {
```
Comment (hackash) [inline js/cc_script.js]: @Mariam13  please use `$.isPlainObject`

File: js/cc_script.js [hunk 5]
```diff
@@ -5,12 +5,15 @@
...
+			options = {};
+		}
 
-		var defaults = {
+	/*================================Configurations==============================*/
+		var defoults = {
```
Comment (hackash) [inline js/cc_script.js]: @Mariam13 please fix grammar issue should be 'defaults'

File: js/cc_script.js [hunk 6]
```diff
@@ -5,12 +5,15 @@
  * Date 09.12.11
  */
 
-(function($){
+(function(window,$){
 	$.fn.ccCarousel = function(options) {
 
-	/*================================Configurations==============================*/
```
Comment (LusineHovsepyan) [inline js/cc_script.js]: .................

File: js/cc_script.js [hunk 7]
```diff
@@ -5,12 +5,15 @@
  * Date 09.12.11
  */
 
-(function($){
+(function(window,$){
```
Comment (hackash) [inline js/cc_script.js]: al least one space should follow to comma

**Pull request merged**